py-utils/src/utils/msg_bus/message_bus/bus/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
```

Comment:
Lets consolidate this code from this folder into message_bus/message_bus.py

Reply:
Consolidated